xeno.cabal

```diff
@@ -1,5 +1,5 @@
 name: xeno
-version: 0.4.2
+version: 0.5
```

Comment:
Not at all committed to a version bump. Will change this to whatever is requested.